### ISSUE TITLE
"Bot is typing" Message persists until bot is restarted

### ISSUE THREAD

**Issue by myurkoski** [Nabrok/gw2-discord-bot#30]
On occasion Discord will say "<Bot_Name> is typing" and the bot never seems to get finished typing. Its hard to say what actually causes this event to occur, as our server has many many users with dozens of channels. When it occurs, a quick bot restart seems to make it right again, and the discord message ges away within moments of shutting the bot down.

Is there any advanced logging I can cat for you, or enable to more thoroughly characterize the issue?

**Comment by Nabrok** [comment]
That would occur if something went wrong between recognizing a command and sending a response.  Logging just goes to the console.  If you're not already I'd suggest running the bot through pm2 which will redirect that output to log files.

**Comment by myurkoski** [comment]
I'm running the bot through systemd on Ubuntu 16.04 LTS, so I've got lovely journalctl outputs if this happens again. 

When it originally happened I had two bots running on the same machine (two different target discords). I killed one and haven't seen this happen again, though its only been a few days. I will keep an eye on it and reopen if it happens again.

**Comment by myurkoski** [comment]
Yeah this continues to occur on rare occasion. Always when a couple people query it in rapid succession.
